src/components/universal/core/Stat/Stat.recipe.ts

```diff
@@ -0,0 +1,97 @@
+import { defineSlotRecipe } from "@pandacss/dev";
+
+export const statRecipe = defineSlotRecipe({
+  className: "stat",
+  description: "The styles for the Stat component",
+  slots: ["root", "label", "value", "helpText"],
+  base: {
+    root: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "start",
+      borderRadius: "md",
+      boxShadow: "sm",
+      bgColor: "accent.default",
+      color: "fg.default",
+      w: "fit",
+      px: 4,
+      py: 2,
+      gap: 0,
+    },
+    helpText: {
+      textStyle: "sm",
+      fontWeight: "light",
+    },
+    label: {
+      textStyle: "md",
+      fontWeight: "normal",
+    },
+    value: {
+      textStyle: "lg",
+      fontWeight: "bold",
```

Comment:
would remove the `fontWeight` style (so that it defaults to `normal`) in order to keep consistent hierarchy with other changes requested

Reply:
What do you mean by consistent hierarchy? I intentionally added all those font weights for different slots. Do you prefer the font weights you listed compared to what I chose, or is there some fancy structure you're trying to go for?

Reply:
I mean that normally, title font (`label` in this case) is given the the bolder weight, even more so than value. This is for consistency when a user is looking around a page, they will first look at "what" the stat is for (i.e. Floor Price) before they look at the value the stat provides. It gives a consistent flow of reading and viewing for the users. I understand wanting emphasis on the value as that is what is driving the stat, but the context for that value needs to be set first, if that makes sense.

Reply:
That makes sense. I followed suit with what Chakra did here: [https://chakra-ui.com/docs/components/stat](url) and went off of that for a good amount of styling decisions, which opposes your flow. What do you think?

Reply:
Interesting! Yeah, I definitely disagree with that decision on their part. But to each their own in design haha. I think that providing context first is super nice, so that users aren't having to go up and down to figure out what the value being provided represents. Just my opinion though! Could also get @coopbri opinion on this matter as well for another take on the matter! Definitely don't need to base it on my thoughts for design theory lol.